### PR TITLE
47 acting erster umsetzungsplan

### PR DESCRIPTION
**Pull request by schwalga**
ca. 6h von mir

### CODE REVIEW

File: doc/research/06_acting_implementation.md
Comment (timdreier) [inline doc/research/06_acting_implementation.md:18]: A reference to the research on acting (05_acting.md) would be quite good.

File: doc/research/06_acting_implementation.md
Comment (timdreier) [inline doc/research/06_acting_implementation.md:50]: Is the speedometer (sensor.other.speedometer) more useful?

https://leaderboard.carla.org/#map-track

File: doc/research/06_acting_implementation.md
Comment (timdreier) [inline doc/research/06_acting_implementation.md:55]: Is there a link to "paf21/1s bug"? It is not directly obvious to me what this is.

**Review by timdreier — CHANGES_REQUESTED** [review]
The file should be moved to 03_research. It is still located in the old folder before we restructured the docs.

Time to review: ~1h

File: doc/research/06_acting_implementation.md
Comment (schwalga) [reply doc/research/06_acting_implementation.md]: Added the link to paf21-2s section on bugs

File: doc/research/06_acting_implementation.md
Comment (schwalga) [reply doc/research/06_acting_implementation.md]: Thanks for the feedback. Speedometer isn't listed in [the list of sensor](https://carla.readthedocs.io/en/latest/ref_sensors/) but should be the obvious choice (fixed in 9970769).

File: doc/research/06_acting_implementation.md
Comment (schwalga) [reply doc/research/06_acting_implementation.md]: Reference added in 9970769

File: doc/research/06_acting_implementation.md
Comment (schwalga) [reply doc/research/06_acting_implementation.md]: Table of contents added. Switched from VSC to PyCharm (fixed in 9970769)

**Comment by timdreier** [comment]
Looks great, but it looks like the path for the 05_acting.md is wrong. Shouldn't it be `docs/03_research/01_acting/05_acting.md` instead of `docs/03_research/03_research /01_acting/05_acting.md`

As far as I understood this is an extended version of the file `docs/03_research/01_acting/02_basics.md` which is already merged. If this is true, this file should also be named `docs/03_research/01_acting/02_basics.md` so that it replaces the merged one.

image

Time to review: ~30min

**Comment by schwalga** [comment]
> Looks great, but it looks like the path for the 05_acting.md is wrong. Shouldn't it be `docs/03_research/01_acting/05_acting.md` instead of `docs/03_research/03_research /01_acting/05_acting.md`
> 
> As far as I understood this is an extended version of the file `docs/03_research/01_acting/02_basics.md` which is already merged. If this is true, this file should also be named `docs/03_research/01_acting/02_basics.md` so that it replaces the merged one.
> 

Thanks, I didn't notice the redundant 03_research directory. -> removed it. (d5effa8)

I don't know how 02_basics.md came into existence, but it should be removed/replaced by 01_acting.md. -> Fixed the enumeration in (d9597b3)

**Comment by timdreier** [comment]
Looks great now, thank you!

Time to approve: ~5min

**Review by timdreier — APPROVED** [review]

**Comment by timdreier** [comment]
From my side yes. Who should responsible to merge a PR?

The one who creates it or the one how reviews it? Maybe we should make this more clear in #35 @JoKircher.

**Comment by JoKircher** [comment]
@timdreier good point there. I will make a suggestion in the current PR #54. I would probably handle this the following way:

If the reviewer has any thing he/she would like to have changed the submits the review as `change required`. If there are no uncertainties the reviewer merges the PR. After a revision of the requested changes the reviewer conducts a second review, if he/she is satisfied with the changes, the PR will be merged by him/her. Long story short, the reviewer who approves the PR should merge. Only approve if there is nothing to change

**Comment by timdreier** [comment]
@JoKircher thanks for clarifying!

**Pull request merged**
